src/Akeneo/Platform/Bundle/UIBundle/Twig/ExternalJavascriptDependenciesExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Platform\Bundle\UIBundle\Twig;
+
+use Akeneo\Platform\Bundle\UIBundle\Provider\ExternalJavascriptDependenciesProvider;
+
+final class ExternalJavascriptDependenciesExtension extends \Twig_Extension
+{
+    private ExternalJavascriptDependenciesProvider $dependenciesProvider;
+
+    public function __construct(ExternalJavascriptDependenciesProvider $dependenciesProvider)
+    {
+        $this->dependenciesProvider = $dependenciesProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('external_javascript_dependencies', [$this, 'getExternalJavascriptDependencies']),
```

Comment:
I think you can add the `is_safe` option to avoid to use it with the `raw` twig filter (`{{ external_javascript_dependencies() | raw }}`)
```suggestion
            new \Twig_SimpleFunction('external_javascript_dependencies', [$this, 'getExternalJavascriptDependencies'], , ['is_safe' => ['html']),
```

Reply:
Thanks for the suggestion, I'll try it